GdscBackend.Tests/TeamsControllerTests.cs

```diff
@@ -0,0 +1,109 @@
+using System.Collections.Generic;
+using System.Linq;
+using FactoryBot;
+using GdscBackend.Controllers.v1;
+using GdscBackend.Database;
+using GdscBackend.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace GdscBackend.Tests
+{
+    public class TeamsControllerTests : TestingBase
+    {
+        private readonly IEnumerable<TeamModel> _testData = _getTestData();
+        
+        public TeamsControllerTests(ITestOutputHelper helper) : base(helper)
+        {
+        }
+
+        [Fact]
+        public async void Delete()
+        {
+            var repos = new Repository<TeamModel>(new TestDbContext<TeamModel>(_testData).Object);
+            var controller = new TeamsController(repos);
+            
+            var actionResult = await controller.Delete(_testData.First().Id);
+
+            Assert.Equal(9,repos.DbSet.Count());
```

Comment:
It's not a good practice to write magic numbers in code (ex: 9). Also, in these tests you want to test controller logic. This means that instead of 9, you can write `(await controller.Get()).Count()` to that the returned elements are the same with the elements from repo